_layouts/speaker.html

```diff
@@ -3,28 +3,70 @@
 ---
 
 {% assign speaker = page %}
+{% assign talk = site.talks | where: "talk_id", speaker.talk_id | first %}
 
-<article class="speaker">
-  <div class="speaker__info">
-    {% include responsive-image
-         path="/assets/images/speakers"
-         basename=speaker.speaker_id
-         srcset="small.jpg 1x, medium.jpg 2x" %}
-    <h1 class="speaker__name">{{ speaker.name }}</h1>
-    <div class="speaker__bio">
-      {{ speaker.bio | newline_to_br }}
+<script>
+  console.log({{speaker|jsonify}});
+  console.log({{talk|jsonify}});
+</script>
+
+<header class="header bg--off-white">
+  <div class="container">
+    <div class="col-xs-8 col-6 col-sm-offset-2">
+      {% include responsive-image
+          path="/assets/images/speakers"
+          basename=speaker.speaker_id
+          srcset="small.jpg 1x, medium.jpg 2x" %}
+    </div>
+
+    <div class="col-xs-4">
+      <h1 class="pt3">
+        {{ speaker.name }}
+      </h1>
     </div>
-    <ul class="speaker__link-list">
-      {% for link in speaker.links %}
-        <a class="speaker__link" href="{{ link.url }}">{{ link.text }}</a>
-      {% endfor %}
-    </ul>
   </div>
-  {% assign talk = site.talks | where: "speaker_id", speaker.speaker_id | first%}
-  <section class="speaker__talk talk">
-    <h1 class="talk__title">{{ talk.title }}</h1>
-    <div class="talk__summary">
-      {{ talk.summary | newline_to_br }}
+</header>
+
+<main>
+  <section>
+    <div class="container">
+      <div class="col-xs-12 col-sm-8 col-sm-offset-2">
+        <div class="box">
+          <article class="speaker">
+            <div class="speaker__info">
+              <div class="speaker__bio">
+                <p class="intro">{{ speaker.intro | newline_to_br }}</p>
+              </div>
+              <div class="speaker__content">
+                {{ speaker.content }}
+              </div>
+
+              <div class="speaker__link-list">
+                {% for link in speaker.links %}
+                  <a class="speaker__link" href="{{ link.url }}">{{ link.text }}</a>
+                {% endfor %}
+              </div>
+            </div>
+
+            <!--
```

Comment:
apparently we won't need this if the editorial text for the speaker will address the talk-outline.